src/SSRv1/templates/bulk/import.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/** @var \WEEEOpen\Tarallo\User $user */
+/** @var array|null $imports */
+$imports = $imports ?? null;
+$this->layout('main', ['title' => 'Import from Peracotta', 'user' => $user, 'currentPage' => 'bulk import']);
+?>
+<div class="row">
+	<div class="col">
+		<table class="table table-borderless stats">
+			<caption>Bulk imports from PERACOTTA</caption>
+			<thead class="thead-dark">
+			<tr>
+				<th scope="col">BulkIdentifier</th>
+				<th scope="col">Time</th>
+				<th scope="col">User</th>
+				<th scope="col">Type</th>
+				<th style="text-align:center" scope="col">Actions</th>
+			</tr>
+			</thead>
+			<tbody>
+				<?php $id=0; foreach($imports as $import): ?>
+					<tr>
+						<td class="align-middle"><?php echo $import['BulkIdentifier']?></td>
```

Comment:
```suggestion
						<td class="align-middle"><?= $this-e($import['BulkIdentifier']) =></td>
```
To this line and the ones below, add `$this-e` to escape them, we could have users sending `<script>alert("XSS")</script>` as the BulkIdentifier or in other fields. The `<?=` look cleaner and we're already using them everywhere else anyway asd